.vitepress/config.js

```diff
@@ -105,6 +105,8 @@ gtag('config', 'UA-24637628-7');
 			dark: '/logo-dark.svg',
 		},
 		nav: [
+			{ text: 'Docs', link: 'http://localhost:5173/' },
+			{ text: 'Cookbook', link: 'http://localhost:5173/cookbook/add-a-recipe', activeMatch: '/cookbook/' },
```

Comment:
Best to keep these URLs relative as they won't be changing domain i believe. That way the links will work too in any deployment.
```suggestion
			{ text: 'Docs', link: '/' },
			{ text: 'Cookbook', link: '/cookbook/add-a-recipe', activeMatch: '/cookbook/' },
```